packages/application-generic/src/health/workflow-queue.health-indicator.ts

```diff
@@ -19,8 +19,9 @@ export class WorkflowQueueServiceHealthIndicator extends HealthIndicator {
 
   async isHealthy(): Promise<HealthIndicatorResult> {
     const isReady = this.workflowQueueService.isReady();
+    const isPaused = this.workflowQueueService.isPaused();
```

Comment:
This is a promise. You need to await it.

Reply:
True, strange that I did not get any warning about it in the editor